### PR TITLE
Application deploy

### PR DESCRIPTION
**Pull request by RolandAmini**
For this milestone I made my website available online.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+- **Add header section**
+- **Add Nav bar**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-  https://rolandamini.github.io/portfolio-mobile/(#)
```
Comment (V-Blaze) [inline README.md]: - [ ] You have done great so far :+1: :100: But there are some changes you need to make before moving to the next milestone. 
- [ ] When I click on the live link in your readme, it takes me to an error page as shown in the screenshot below. I'll suggest you update this. :+1: :+1: 

- - Screenshot:
  
Screenshot from 2023-07-17 14-20-29

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @RolandAmini , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Readme looks professional✔
-  Descriptive pull request title and summary👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @RolandAmini  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Used Github pages💯
-  Live link added to Readme👍
-  Deployed project works great🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RolandAmini** [comment]
thank you @V-Blaze

**Pull request merged**